src/query/config/src/config.rs

```diff
@@ -2930,6 +2935,26 @@ pub struct DiskCacheConfig {
     pub sync_data: bool,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Args, Default)]
+#[serde(default, deny_unknown_fields)]
+pub struct SpillConfig {
+    /// Path of spill to local disk. disable if it's empty.
+    #[clap(
+        long,
+        value_name = "VALUE",
+        default_value = "./.databend/_cache/temp/_query_spill"
```

Comment:
Temporarily try to modify it to `./.databend/_cache` and test it on databend cloud.